tests/data/Napkin_data_generator.py

```diff
@@ -0,0 +1,50 @@
+seed = 1
+num_samples = 1000
+import numpy as np
+import pandas as pd
+from ananke.estimation import CausalEffect
+from ananke.graphs import ADMG
+#Generate observational data for napkin
+def generate_obs_data_for_napkin(seed = seed, num_samples = num_samples):
+  np.random.seed(seed)
+  #U1 is the latent variable that is a common cause of W and X
+  U1 = np.random.normal(loc = 3, scale = 1, size = num_samples)
```

Comment:
would be better to create a random generator object from the seed then use that directly: https://numpy.org/doc/stable/reference/random/generator.html